### ISSUE TITLE
loader option causes hangs with some games

### ISSUE THREAD

**Issue by medievil1** [Cxbx-Reloaded/Cxbx-Reloaded#1919]
Max Payne:
When using loader option and attempting to resume, game never continues/loads
When not using loader option, resumes works normally
Splinter Cell DA:
when using loader option, loading screen hangs and game never starts.
when not using loader option, game loads and starts

(Will add more as I run across them)

**Comment by RadWolfie** [comment]
Is your "debug output (kernel)" set to console while using loader option? If so, try again with "debug output (kernel)" disabled.

If titles start to work again with loader, excluding "debug output (kernel)" option. Then we do have a ticket opened for known issue. See issue #1882

**Comment by medievil1** [comment]
no, it is disabled......since the introduction of the loader there are a lot of weird quirks like starting emulation only to crash back ui within 3 seconds, same a second time, then 3rd time it running normally...but that happens loader or no loader and happens randomly no particular game...I didn't include it since it isn't directly loader related but DID start around the time the loader was added, so whatever is causing that could also be glitching the loader

**Comment by medievil1** [comment]
got more information, hadn't thought to look... it leaves an error in event viewer when it starts and closes back to UI immediately. I tried to start a xbe, took 3 times, have this same error twice, 2 seconds apart as it closed and I started it again 2 seconds later 

> Faulting application name: cxbx.exe, version: 0.0.0.0, time stamp: 0x5ee29a83
Faulting module name: KERNELBASE.dll, version: 10.0.18362.815, time stamp: 0x67005c2a
Exception code: 0xc0000409
Fault offset: 0x00114402
Faulting process id: 0x1494
Faulting application start time: 0x01d640340d3b0bbd
Faulting application path: Y:\cxbxsource\test\Cxbx-Reloaded\build\bin\Release\cxbx.exe
Faulting module path: C:\Windows\System32\KERNELBASE.dll
Report Id: 3ead5df8-ce6b-4d15-8eac-e77b8279652b
Faulting package full name: 
Faulting package-relative application ID:

**Comment by gandalfthewhite19890404** [comment]
Here are some dumps, first - crashed, second (emulation started after first one crashed) - works, maybe it could be helpful:
AFL Live 2004, 2006 FIFA World Cup, NHL 07, Race Driver 3.
[some_dumps.zip](https://github.com/Cxbx-Reloaded/Cxbx-Reloaded/files/4852555/some_dumps.zip)
